.github/SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Policy
+
+## Supported Versions
+
+Unless otherwise noted, MCP packages and projects support the latest version of Meteor.
+As of the time of writing this in September 2023 it is Meteor 2.13 and, in general the 2.x series.
+Support for Meteor 1.x is not guaranteed and at best can include Meteor 1.12.1.
+At the present time, we are also preparing to support Meteor 3.0 which is in development.
+
+| Meteor Version | Supported          |
+| -------------- | ------------------ |
+| 3.0            | 🚧                 |
+| > 2.0          | ✅                 |
+| < 1.12         | ❌                 |
```

Comment:
I couldn't locate any information regarding the Meteor versions with which this package has undergone testing. I also reviewed the earlier versions, namely versions 1 and 2. It would be beneficial to include information about the supported Meteor version for these older releases, even though they are no longer officially supported.

Reply:
@timsun28 sorry to bother you with this, this was intended for @StorytellerCZ

Reply:
@jankapunkt true, I think we can leave it for now and in a next round update it to `<=`.